engine/src/main/java/org/hibernate/search/engine/backend/types/Norms.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.backend.types;
+
+/**
+ * Whether index time scoring information for the field should be stored or not.
+ * <p>
+ * Not storing the norms can save a considerable amount of memory.
+ * Storing them can make scoring queries faster.
```

Comment:
With `Storing them` I meant `enabling` and not `disabling` the norms. :P 
Anyway, let's change it as you suggested. In your version it's more clear the dichotomy of the two sentences.

Reply:
Right, that was wrong.

Even then, enabling norms should not speed up queries, since norms won't be computed at query time when they are disabled. At least there's no *obvious* reason to gain performance from that, as there's no extra cost at query time when norms are disabled.